src/ostorlab/agent/message/proto/v3/fingerprint/file/android/backend/backend.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.fingerprint.file.android.backend;
+
+message location {
+  optional string host = 1;
```

Comment:
The proto is not respecting the hiarachy, I think it is missing the fields from the parent protos.